network.go

```diff
@@ -75,6 +78,53 @@ func (n *network) Type() string {
 	return n.driver.Type()
 }
 
+func (n *network) Key() []string {
+	return []string{"network", string(n.id)}
+}
+
+func (n *network) Value() []byte {
+	b, err := json.Marshal(n)
```

Comment:
@mrjana no. the methods `MarshalJSON` and `UnmarshalJSON` belongs to the `RawMessage` type defined in json package : http://golang.org/pkg/encoding/json/#RawMessage
When the `json.Marshal` is invoked, the json package calls back the `MarshalJSON` defined in the object (which implements RawMessage).